### PR TITLE
make -f files executable

### PR DESCRIPTION
**Pull request by marcmengel**
In case they're uploading a script...

### CODE REVIEW

**Review by shreyb — COMMENTED** [review]
LGTM, but I'm going to test it before approving.

File: templates/simple/simple.sh [hunk 1]
```diff
@@ -210,6 +210,7 @@ export PATH="${PATH}:."
 # -f files for input
 {%for fname in input_file%}
 ${JSB_TMP}/ifdh.sh cp -D {{fname}} ${CONDOR_DIR_INPUT}
+chmod u+x ${CONDOR_DIR_INPUT}/{{fname}}
```
Comment (shreyb) [inline templates/simple/simple.sh:213]: So there's a mistake here.  As is, we'd be trying to do something like 

`chmod u+x ${CONDOR_DIR_INPUT}/cvmfs/fifeuser1.opensciencegrid.org/sw/mu2e/4735aadedec8938e9ba7d03829af68a5d03429667f8dfcce5d8e95df29929dff/myexecutable`, where what we want is

`chmod u+x ${CONDOR_DIR_INPUT}/myexecutable`

I think this line needs to be:

`chmod u+x ${CONDOR_DIR_INPUT}/{{fname|basename}}`

Does that sound right?

**Review by shreyb — CHANGES_REQUESTED** [review]

File: templates/simple/simple.sh (hunk 1 above)
Comment (marcmengel) [reply templates/simple/simple.sh:213]: Yep.

**Review by shreyb — APPROVED** [review]
LGTM.

**Pull request merged**